be/src/tools/meta_tool.cpp

```diff
@@ -97,11 +97,14 @@ std::string get_usage(const std::string& progname) {
           "--root_path=/path/to/storage/path --tablet_id=tabletid "
           "[--schema_hash=schemahash]\n";
     ss << "./meta_tool --operation=delete_meta --tablet_file=file_path\n";
+    ss << "./meta_tool --operation=delete_table_meta --table_id=table_id\n";
     ss << "./meta_tool --operation=delete_rowset_meta "
```

Comment:
why introduce a new operation? Is this possible?
```
--operation=delete_meta  --tablet_id|--table_id
```

Reply:
> why introduce a new operation? Is this possible?
> 
> ```
> --operation=delete_meta  --tablet_id|--table_id
> ```

Yes, this can be done. But since the delete_table_meta operation will delete all the tablets meta of the table, I am worried that mixing it with delete_meta may cause a misoperation.

Reply:
table_id and tablet_id will not be the same, so it should be safe

Reply:
> table_id and tablet_id will not be the same, so it should be safe

done